offerzen/global/employers/employers@1.0.js

```diff
@@ -0,0 +1,361 @@
+window.$loaded(function (window, document, $, undefined) {
+  ;(function () {
+    window.$parsleyLoaded = function (cb) {
+      setTimeout(() => {
+        if (window.Parsley && window.ParsleyValidator) {
+          cb(window, document, parsley, undefined)
+          return
+        }
+        $parsleyLoaded(cb)
+      }, 50)
+    }
+  })()
+
+  function getRoleTypes(formData, formProperties) {
+    let fields = [
+      'role_type[In-office]',
+      'role_type[Fully-remote]',
+      'role_type[Hybrid]',
+    ]
+    let role_types = []
+
+    for (let key in formProperties) {
+      if (fields.includes(key) && formProperties[key] === 'on') {
+        if (key === 'role_type[Fully-remote]') {
+          role_types.push(
+            key.replace('role_type[', '').replace(']', '').replace('-', ' ')
+          )
+        } else {
+          role_types.push(key.replace('role_type[', '').replace(']', ''))
+        }
+        delete formProperties[key]
+      }
+    }
+
+    return role_types
+  }
+
+  function tracking() {
+    dataLayer.push({
+      event: 'Company Lead Form Submitted ',
+      action: 'Lead Form Submitted',
+      label: 'Company Sign Up / Employer Landing Page',
+      category: 'Core',
+      source: 'Demand Sign Up',
+    })
+  }
+
+  window.onSubmitCompanyLeadForm = function (token, e) {
+    window.$parsleyLoaded(function (window, document, parsley) {
+      form = $('#wf-Company-Lead-Form')
```

Comment:
```suggestion
      const form = $('#wf-Company-Lead-Form');
```
Otherwise it gets added to `window`